examples/odata-cli/package.json

```diff
@@ -9,7 +9,7 @@
         "start": "node -r dotenv/config dist/index.js dotenv_config_path=./.env/env.properties",
         "build": "pnpm clean && tsc",
         "watch": "tsc --watch",
-        "clean": "rimraf dist coverage",
+        "clean": "rimraf dist coverage tsconfig.tsbuildinfo",
```

Comment:
Why remove `tsconfig.tsbuildinfo`? This file is being generated because `composite` is true in `tsconfig`.
Removing it every time defeats the purpose of incremental builds.

Reply:
I will keep it cleaned up for now, since for some reason it is blocking the dist folder to be generated. I can investigate why this is happening later.